toolchain/check/check_fuzzer.cpp

```diff
@@ -10,6 +11,18 @@
 
 namespace Carbon::Testing {
 
+static InstallPaths install_paths;
+
+// NOLINTNEXTLINE(readability-non-const-parameter): External API required types.
+extern "C" auto LLVMFuzzerInitialize(int* argc, char*** argv) -> int {
+  std::string exe_path;
+  if (*argc >= 1) {
+    exe_path = FindExecutablePath((*argv)[0]);
```

Comment:
```suggestion
  if (*argc >= 1) {
    std::string exe_path = FindExecutablePath((*argv)[0]);
```

Would this make sense to be clear on the lifetime?

Reply:
No idea what I was thinking here, sorry. =] But this is also obviated a bit with new code pattern.